zbl_py/src/lib.rs

```diff
@@ -122,23 +125,27 @@ impl Capture {
         window_name: Option<&str>,
         window_handle: Option<i32>,
         display_id: Option<i32>,
-        capture_cursor: Option<bool>,
+        is_cursor_capture_enabled: Option<bool>,
+        is_border_required: Option<bool>,
         cpu_access: Option<bool>,
     ) -> PyResult<Self> {
-        let capture_cursor = capture_cursor.unwrap_or(false);
+        let is_cursor_capture_enabled = is_cursor_capture_enabled.unwrap_or(false);
+        let is_border_required = is_border_required.unwrap_or(false);
```

Comment:
I think we should change the default to be `true` (and in the Python version as well) to mirror the WGC API.